docs/testplan/pac/Port_Access_Control.md

```diff
@@ -0,0 +1,562 @@
+
+
+### 3.2.3 Verify 802.1x with multi-host mode and verify all users on the port are allowed after first-user gets authenticated.
+
+| **Test ID**    | **PAC_FUNC_003**                         |
+| -------------- | :--------------------------------------- |
+| **Test Name**  | **Verify 802.1x with multi-host mode and verify all users on the port are allowed after first-user gets authenticated.** |
+| **Test Setup** | **Topology1**                            |
+| **Type**       | **Functional**                           |
+| **Steps**      | 1. Enable 802.1x authentication on interface.<br/>2. Configure RADIUS server on the authenticator.<br/>3. Enable authentication host mode to multi-host mode<br/>4. Initiate Authentication from 802.1x supplicant and Verify Authenticator encapsulates EAP packets and sends it to RADIUS server<br/>6. Verify Authenticator moves client to Authenticated state after sending EAP success, once authenticator receives Access-accept from RADIUS server<br/>7. Verify "show authentication clients all" to see the client state<br/>8. Verify client's MAC FDB entry is populated once the client sends traffic after client gets authenticated<br/>9. Verify untagged and tagged traffic gets forwarded from the client's RADIUS assigned VLAN if not port's untagged VLAN after it gets authenticated on the port, check non matching tagged traffic is not allowed.<br/>10. Verify all the subsequent Clients traffic connected to port also gets access and check MAC address gets updated for all those clients<br/>11. Try to authenticate another client and check it is not successful as the port is in multi-host mode.<br/>12. Logoff client1 and verify all the clients connected to the port gets blocked<br/> |
```

Comment:
Step 11. Authentication is not required.

Reply:
Corrected statement as per review comment.